### PR TITLE
METRON-2128: LEEF config file is missing in RPM spec file

### PR DESCRIPTION
**Pull request by simonellistonball**
## Contributor Comments
The default config sample used in the LEEF parser integration test, and provided as a sample starting point was not included in the RPM SPEC file. 

This PR fixes this omission and cleans up whitespace formatting the parsers to ensure better style guide compliance (only actual changes in the java files here are whitespace)

To Test: verify build succeeds with `mvn clean package -DskipTests -Pbuild-rpms`

## Pull Request Checklist

Thank you for submitting a contribution to Apache Metron.  
Please refer to our [Development Guidelines](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=61332235) for the complete guide to follow for contributions.  
Please refer also to our [Build Verification Guidelines](https://cwiki.apache.org/confluence/display/METRON/Verifying+Builds?show-miniview) for complete smoke testing guides.  

In order to streamline the review of the contribution we ask you follow these guidelines and ask you to double check the following:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? If not one needs to be created at [Metron Jira](https://issues.apache.org/jira/browse/METRON/?selectedTab=com.atlassian.jira.jira-projects-plugin:summary-panel).
- [ ] Does your PR title start with METRON-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.
- [ ] Has your PR been rebased against the latest commit within the target branch (typically master)?

### For code changes:
- [x] Have you included steps to reproduce the behavior or problem that is being changed or addressed?
- [x] Have you included steps or a guide to how the change may be verified and tested manually?
- [x] Have you ensured that the full suite of tests and checks have been executed in the root metron folder via:
  ```
  mvn -q clean integration-test install && dev-utilities/build-utils/verify_licenses.sh 
  ```

- [x] Have you written or updated unit tests and or integration tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] Have you verified the basic functionality of the build by building and running locally with Vagrant full-dev environment or the equivalent?

### For documentation related changes:
- [x] Have you ensured that format looks appropriate for the output in which it is rendered by building and verifying the site-book? If not then run the following commands and the verify changes via `site-book/target/site/index.html`:

  ```
  cd site-book
  mvn site
  ```

- [ ] Have you ensured that any documentation diagrams have been updated, along with their source files, using [draw.io](https://www.draw.io/)? See [Metron Development Guidelines](https://cwiki.apache.org/confluence/display/METRON/Development+Guidelines) for instructions.

#### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.
It is also recommended that [travis-ci](https://travis-ci.org) is set up for your personal repository such that your branches are built there before submitting a pull request.

### CODE REVIEW

**Comment by ottobackwards** [comment]
the leef configuration does not go in -common, can you move it please?

**Comment by simonellistonball** [comment]
Agreed, but that what the integration tests look for, so that’s where it needs to live to get the integration tests required by you comments on the other PR. Can you suggest where it should go, and how we should change the integration test approach?

**Comment by mmiklavc** [comment]
> Agreed, but that’s what the integration tests look for, so that’s where it needs to live to get the integration tests required by you comments on the other PR. Can you suggest where it should go, and how we should change the integration test approach?

It actually checks for zookeeper config in both `metron-parsing/metron-parsers` and `metron-parsing/metron-parsers-common` - https://github.com/apache/metron/blob/22cc622d6f08cfa77c06dea2df7f4c56edc1862e/metron-platform/metron-parsing/metron-parsers-common/src/test/java/org/apache/metron/parsers/integration/ParserIntegrationTest.java#L67. I didn't actually realize that until I dug a bit. Echoing our earlier thoughts, I would like to simplify this a bit at some point so that the concerns aren't jumbled.

That being said, the leef config should go here - https://github.com/apache/metron/tree/22cc622d6f08cfa77c06dea2df7f4c56edc1862e/metron-platform/metron-parsing/metron-parsers/src/main/config/zookeeper/parsers

File: metron-deployment/packaging/docker/rpm-docker/SPECS/metron.spec [hunk 1]
```diff
@@ -164,6 +164,7 @@ This package installs the Metron Parser Common files
 %{metron_home}/config/zookeeper/parsers/jsonMapWrappedQuery.json
 %{metron_home}/config/zookeeper/parsers/syslog3164.json
 %{metron_home}/config/zookeeper/parsers/syslog5424.json
+%{metron_home}/config/zookeeper/parsers/leef.json
```
Comment (mmiklavc) [inline metron-deployment/packaging/docker/rpm-docker/SPECS/metron.spec]: Don't forget to move the config file in metron.spec also

**Comment by mmiklavc** [comment]
@simonellistonball - I opened a PR against your PR moving the leef config in metron.spec. I ran the build locally and checked that the RPM's build via `time mvn clean package -DskipTests -Pbuild-rpms`

Fwiw, here were the results of the timing - `1111.11s user 154.80s system 56% cpu 37:13.88 total`

**Comment by justinleet** [comment]
@simonellistonball Would you mind bouncing Travis? Seems like an unrelated failure.

**Comment by simonellistonball** [comment]
> @simonellistonball Would you mind bouncing Travis? Seems like an unrelated failure.

Not sure what the problem is... all the travis checks passed on this.

**Comment by justinleet** [comment]
@mmiklavc Are you good with this? I'm going to abstain since I mostly saw a Travis issue that I probably commented on the wrong PR for, based on the response from @simonellistonball

**Comment by mmiklavc** [comment]
@justinleet Everything looks good, except we're missing an update to the changelog file here - https://github.com/apache/metron/blob/02bd729bf349095e4fc7054d647dd6b903ac8bb7/metron-deployment/packaging/docker/rpm-docker/SPECS/metron.spec#L688. It looks like we also missed the update from a PR I submitted as well. I'm going to open a separate PR to fix this so we can get master working again.

+1 by inspection

**Comment by mmiklavc** [comment]
https://issues.apache.org/jira/browse/METRON-2144

**Pull request closed without merging**